### DISCUSSION TITLE
How can i execute singal receive many times

### DISCUSSION THREAD

**Discussion by tuohaibei** [elsa-workflows/elsa-core#4528] [Q&A] [answered]
I have customized the activity. After the current activityis executed, I do not want the current activity to end. I want the current instance to end after executing the current activity multiple times. Is there a good solution?
when i send signal recived,report activity could execute,i want i could execute signal received many times  and trigger report activity and after that,the instance could finish.
![image](https://github.com/elsa-workflows/elsa-core/assets/5082315/aac43580-8740-466f-8031-100f8057dac7)

**Comment by tuohaibei** [comment]
could anybody could help me?

**Reply by tuohaibei** [reply]
？

**Comment by sfmskywalker** [comment, marked as answer]
Hi @tuohaibei , you can connect the Report activity back to the Signal Received activity. If you’re not sure how to do that, this is documented here: https://v2.elsaworkflows.io/docs/dashboard/designer

**Reply by tuohaibei** [reply]
thanks very much,i have fixed it according you provide document.
